pep-0458.txt

```diff
@@ -974,6 +974,18 @@ of a package at a specific version, they can be handled by TUF with techniques
 like implicit key revocation and metadata mismatch detection [2].
 
 
+Managing Future Changes to the Update Process
+=============================================
+
+If breaking changes are made to the update process, PyPI should implement these
+changes without disrupting existing clients. To do so, PyPI should maintain
+previous versions of metadata so that clients can access the version of metadata
```

Comment:
This is too vague.  We don't have enough specifics now.  My feeling is it should be written in a less ambiguous way (referring to look at the TAP for data storage), even if it says less here.

Reply:
Agreed